pkg/apis/sensor/v1alpha1/types.go

```diff
@@ -242,8 +242,23 @@ type DataFilter struct {
 	// Strings are taken as is
 	// Nils this value is ignored
 	Value []string `json:"value" protobuf:"bytes,3,rep,name=value"`
+	// Comparator compares the event data with a user given value.
+	// Can be ">=", ">", "=", "<", or "<=".
+	// Is optional, and if left blank treated as equality "=".
+	Comparator Comparator `json:"comparator,omitempty" protobuf:"bytes,4,opt,name=comparator"`
 }
 
+type Comparator string
```

Comment:
Can you move the type declaration to the top with other types? just to keep it consistent. Also add a comment on the `type Comparator string`.

Reply:
updated